city_scrapers/spiders/alle_port_authority.py

```diff
@@ -0,0 +1,179 @@
+# -*- coding: utf-8 -*-
+import unicodedata
+from datetime import datetime, timedelta
+
+from dateutil.parser import parse
+from lxml import html
+
+from city_scrapers.constants import BOARD, COMMITTEE
+from city_scrapers.spider import Spider
+
+
+class AllePortAuthoritySpider(Spider):
+    name = 'alle_port_authority'
+    agency_name = 'Port Authority of Allegheny County'
+    timezone = 'America/New_York'
+    allowed_domains = ['www.portauthority.org']
+    start_urls = [
+        'https://www.portauthority.org/paac/CompanyInfoProjects/'
+        'BoardofDirectors/MeetingAgendasResolutions.aspx'
+    ]
+    event_year = datetime.now().year
+
+    def _get_address(self, response):
+        address = (response.xpath('//table[1]//span/text()').extract()[0])
+        return address
+
+    def _build_datatable(self, response):
+        alist_tbody = (response.xpath('//table[1]/tbody//td').extract())
+
+        atable = []
+        arow = []
+
+        for item in alist_tbody:
+            tree = html.fragment_fromstring(item)
+            text = tree.text_content()
+
+            url = tree.xpath('//a/@href')
+            find_att_b = tree.xpath('//b/text()|//strong/text()')
+            if len(find_att_b) >= 1:
+                continue
+            if url:
+                arow.append('{name}: {url}'.format(name=text, url=url[0]))
```

Comment:
We should still use a dictionary instead of a formatted string here to keep things more clear

Reply:
This is just for internal usage. I am building a 2D list from a table from HTML. The final url will be a dictionary. It is just regenerating a html table as a 2D list.